### PR TITLE
Change group leader script

### PR DESCRIPTION
**Pull request by dexterlb**
This script has been originally written by @wpferguson as a [comment](https://github.com/darktable-org/darktable/issues/7030#issuecomment-733458027) on [darktable issue #7030](https://github.com/darktable-org/darktable/issues/7030).

I have been using it for some time without problems, and several people have expressed interest in including it in `lua-scripts`.

I am willing to maintain the script for the foreseeable future.

I am not sure that creating a PR like this for code that was originally written by someone else (especially adding a copyright notice under my own name) is the correct course of action, so I am sorry if it is not.

### CODE REVIEW

File: contrib/change_group_leader.lua [hunk 1]
```diff
@@ -0,0 +1,168 @@
+--[[
+  Passport cropping guide for darktable
+
```
Comment (wpferguson) [inline contrib/change_group_leader.lua:3]: Add
`copyright (c) 2020 Bill Ferguson <wpferguson@gmail.com>`

File: contrib/change_group_leader.lua [hunk 2]
```diff
@@ -0,0 +1,168 @@
+--[[
+  Passport cropping guide for darktable
```
Comment (wpferguson) [inline contrib/change_group_leader.lua:2]: Change from `Passport cropping guide for darktable` to `change group leader`

File: contrib/change_group_leader.lua [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+
+--[[
+  Original author: github.com/wpferguson
+]]
```
Comment (wpferguson) [inline contrib/change_group_leader.lua:22]: You can delete this, the copyright comment above takes care of it, but thanks for the attribution.

**Review by wpferguson — COMMENTED** [review]
I looked back at my initial commit message and the only thing really  missing is i18n for the strings.  If you don't want to do it, that's fine, I'll do it later.

**Comment by wpferguson** [comment]
I'd forgotten about this script.  Thanks for picking it up and submitting it.

**Comment by dexterlb** [comment]
What is there left to be done for i18n? I see that all messages are tagged

Or do you mean the actual translations?

**Comment by wpferguson** [comment]
translation involves adding

```
local gettext = dt.gettext
-- Tell gettext where to find the .mo file translating messages for a particular domain
gettext.bindtextdomain(MODULE,dt.configuration.config_dir.."/lua/locale/")

local function _(msgid)
    return gettext.dgettext(MODULE, msgid)
end
```

and then wrapping strings that appear in the user interface with

`_("a string to be translated")`

Strings used for debugging (`dt.print_log()`) don't get translated.

**Comment by wpferguson** [comment]
@dexterlb you can leave your copyright notice in the file also, since you did make additions

**Pull request merged**